### ISSUE TITLE
Videos In Very Low Resolution for First 5 Seconds

### ISSUE THREAD

**Issue by CathalCraft** [YePpHa/YouTubeCenter#2019]
YoutubeCenter Dev Build #533
Firefox 40.0.2
 
For the past few days I've noticed that YouTube videos are very low resolution for the first 5 seconds of every video I play. I reset YoutubeCenter settings to default and the problem still persists.

Example: http://puu.sh/jFxzu/3265edbb81.jpg

**Comment by Yonezpt** [comment]
This is YouTube's new behavior, it usually does this so that the user does not have to wait to watch the video while it is loading the higher quality, plays the lower, smaller and faster quality while the other one is buffering.

**Comment by vule76** [comment]
Try to update Shockwave flash:

![222](https://cloud.githubusercontent.com/assets/7959205/9343030/a0d9afea-45ff-11e5-9b9e-d9ec949e0eb6.jpg)

**Comment by CathalCraft** [comment]
I don't have Shockwave Flash installed.

**Comment by vule76** [comment]
Try to update your Flash player. https://get.adobe.com/flashplayer/

**Comment by Yonezpt** [comment]
Vule, he is using the html5 player, not the flash player. Also the player layout in his screenshot only exists for the html5 player.

**Comment by vule76** [comment]
I use default settings for player in Yt center and I get the same layout after I update the flash player (Shockwave). I don't know then.

**Comment by Yonezpt** [comment]
@vule76 This issue does not happen to everyone or that frequently, the speed connection or server locations might affect this, or more conditions that are not known to us yet. I've seen it often with recently released videos of channels that are not that popular (usually these don't get their videos copied to as many servers as popular channel videos so they tend to download slower even if you have high speeds).

And if you are getting the new player layout then you are using the html5 player, it is the only player that has the new layout, YouTube never released it for the flash player and I doubt they will ever release it since they intend to stop using it.

I just confirmed this by forcing the fheir flash player and they are still using the old layout.

![ss](https://cloud.githubusercontent.com/assets/8498057/9345583/094a2332-4602-11e5-9b51-2f575f2024d5.png)

Right click on your player and you will not see the same adobe flash player menu.

**Comment by vule76** [comment]
Yes, you are right, I didn't know that. Html5 was big problem with me because it was freezing allot so I forced flash but now is all in htlm5 and it's all ok. Sorry for spam then.
